packages/mdc-typography/README.md

```diff
@@ -78,18 +79,19 @@ If you want to set the typographic style of an element, which is not a Material
 CSS Class | Description
 --- | ---
 `mdc-typography` | Sets the font to Roboto
-`mdc-typography--display4` | Sets font properties as Display 4
-`mdc-typography--display3` | Sets font properties as Display 3
-`mdc-typography--display2` | Sets font properties as Display 2
-`mdc-typography--display1` | Sets font properties as Display 1
-`mdc-typography--headline` | Sets font properties as Headline
-`mdc-typography--title` | Sets font properties as Title
-`mdc-typography--subheading2` | Sets font properties as Subheading 2
-`mdc-typography--subheading1` | Sets font properties as Subheading 1
-`mdc-typography--body2` | Sets font properties as Body 2
+`mdc-typography--headline1` | Sets font properties as Headline 1
```

Comment:
In the first paragraph of this section above, replace the obsolete card example there with one we're actually prescriptive about, e.g. tab and button using `button` style